crates/astria-conductor/src/celestia/mod.rs

```diff
@@ -225,7 +230,13 @@ impl Reader {
                         }
                         Err(TrySendError::Full(block)) => {
                             trace!("executor channel is full; rescheduling block fetch until the channel opens up");
-                            scheduled_block = executor.clone().send_firm_block(block).boxed().fuse();
+                            let executor_clone = executor.clone();
+                            // Using async block returning celestia height so reference height be updated
```

Comment:
Small nit: just mention the need for returning the height. No need to mention the async block (since that's obvious from the next line)